integrationTests/testProcessorNode.go

```diff
@@ -516,11 +516,15 @@ func newBaseTestProcessorNode(args ArgTestProcessorNode) *TestProcessorNode {
 
 	logsProcessor, _ := transactionLog.NewTxLogProcessor(transactionLog.ArgTxLogProcessor{Marshalizer: TestMarshalizer})
 
-	args.RunTypeComponents = components.GetRunTypeComponentsWithCoreComp(&mock.CoreComponentsStub{
-		HasherField:              TestHasher,
-		InternalMarshalizerField: TestMarshalizer,
-		EnableEpochsHandlerField: enableEpochsHandler,
+	rtc := components.GetRunTypeComponentsWithCoreComp(&mock.CoreComponentsStub{
+		HasherField:                 TestHasher,
+		InternalMarshalizerField:    TestMarshalizer,
+		EnableEpochsHandlerField:    enableEpochsHandler,
+		AddressPubKeyConverterField: &testscommon.PubkeyConverterStub{},
 	})
+	runTypeComponents := components.GetRunTypeComponentsStub(rtc)
+	runTypeComponents.AccountParser = &genesisMocks.AccountsParserStub{}
```

Comment:
Can't we use a real stub here (since that is not a stub, it uses real compomenents) so we don't have to set accounts parser afterwards to stub?

Reply:
It was a stub before. Will try with real component

Reply:
some long tests are failed if I don't do this